### ISSUE TITLE
Deccan delight add on shows video error

### ISSUE THREAD

**Issue by bumehta17031947** [Vikassm73/AjaykRepo#132]
Deccan Delight Add on hindi scrappers Desitashan,desiserials and yo desi shows video error could help pluease to resolve it as gujal says it is problem of add on and they raise hands conveying issue is closed.

**Comment by Vikassm73** [comment]
Yes that particular module of addon does not work... I have already done work on that.

**Comment by bumehta17031947** [comment]
Could you send its solution please , Regards

**Comment by Vikassm73** [comment]
There is no solution right now

**Comment by ksadda77** [comment]
> 
In Deccan delight add on, only movie rulz is working. All other options not working from many days.

**Comment by Vikassm73** [comment]
Most of the links are blocked by ISP. In that case either you have to use VPN. Also, you can try Cloudflare DSN. If you are directly connected with the wired line then  it works but in wifi it may or may not work.
